test/unit/test_disco_storage.py

```diff
@@ -128,9 +128,8 @@ def test_create_ebs_snapshot_unencrypted(self):
         self.assertEquals(False, snapshots[0].encrypted)
         self.assertEquals('mock_productline', snapshots[0].tags['productline'])
 
-    @mock_ec2
-    def test_take_snapshot(self):
-        """Test taking a snapshot of an attached volume"""
+    def _test_take_snapshot_create_volume(self):
```

Comment:
If this is a helper method instead of an actual unit test then it probably shouldn't start with `test`. How about just call it `_create_volume`